youtubesearchpython/core/channel.py

```diff
@@ -8,20 +8,35 @@
 from youtubesearchpython.core.componenthandler import getValue, getVideoId
 
 
-
 class ChannelCore(RequestCore):
-    def __init__(self, channelId: str):
+    def __init__(self, channel_id: str, request_params: str):
         super().__init__()
-        self.browseId = channelId
+        self.browseId = channel_id
+        self.params = request_params
+        self.result = {}
+        self.continuation = None
 
     def prepare_request(self):
         self.url = 'https://www.youtube.com/youtubei/v1/browse' + "?" + urlencode({
             'key': searchKey,
             "prettyPrint": "false"
         })
         self.data = copy.deepcopy(requestPayload)
-        self.data["params"] = "EgVhYm91dA%3D%3D"
-        self.data["browseId"] = self.browseId
+        if not self.continuation:
+            self.data["params"] = self.params
+            self.data["browseId"] = self.browseId
+        else:
+            self.data["continuation"] = self.continuation
+
+    def playlist_parse(self, i) -> dict:
+        return {
+            "id": getValue(i, ["playlistId"]),
+            "thumbnails": getValue(i, ["thumbnail", "thumbnails"]),
+            "title": getValue(i, ["title", "runs", 0, "text"]),
+            "videoCount": getValue(i, ["videoCountShortText", "simpleText"]),
+            "viewCount": getValue(i, ["v", 1, "simpleText"]),
+            "lastEdited": getValue(i, ["publishedTimeText", "simpleText"]),
```

Comment:
Have you actually seen these in a playlist response? I haven't so far

Reply:
I've seen it in YouTube's JSON, but cannot confirm that I correctly copy-pasted the paths. I seem to have forgotten to remove `viewCount`. It's definitely invalid. Will fix everything tomorrow (or you may now, if you want to).

Reply:
I just confirmed that `lastEdited` actually appears, and removed `viewCount`